File: webapp/backend/src/main/java/com/computronica/webapp/model/Tema.java
// src/main/java/com/computronica/webapp/model/Tema.java
package com.computronica.webapp.model;

import com.google.cloud.Timestamp;
import lombok.Data;

import java.io.Serializable;

@Data
public class Tema {
    private String id;
    private String asignaturaId;
    private String nombre;
    private String descripcion;
    private boolean estado=true;
    private Timestamp fechaCreacion;

}

File: CompuTronica/app/src/main/java/com/example/computronica/Model/Tema.kt
package com.example.computronica.Model

import com.google.firebase.Timestamp
import java.io.Serializable

data class Tema(
    val id: String = "",
    val asignaturaId: String = "",
    val nombre: String = "",
    val descripcion: String = "",
    val estado: Boolean = true,
    val fechaCreacion: Timestamp? = null
) : Serializable